cli/cmd/cluster.go

```diff
@@ -339,18 +457,22 @@ func newGetCredentialsCommand() *cobra.Command {
 				return fmt.Errorf("could not extract current context from config")
 			}
 
-			context.AuthInfo = opts.serviceAccountName
+			context.AuthInfo = accountName
 			config.Contexts = map[string]*api.Context{
 				config.CurrentContext: context,
 			}
 			config.AuthInfos = map[string]*api.AuthInfo{
-				opts.serviceAccountName: {
+				accountName: {
 					Token: string(token),
 				},
 			}
 
 			cluster := config.Clusters[context.Cluster]
 
+			if opts.remoteClusterServer != "" {
```

Comment:
@zaharidichev can you explain this to me?

Reply:
`--cluster-server` is for overriding the k8s api server url to avoid this:
https://github.com/olix0r/l2-k3d-multi/blob/f64de67f0a7766b04f0b288b8e0128363fd2b2ed/link.sh#L12-L19

Reply:
Oh, this is the address of the target cluster's kubernetes API server.  Gotcha.

Can we maybe call this `--api-server-address` or something?  I had no idea what `cluster-server` meant.  The flag description "Custom remote cluster domain" is also incorrect, I think.